### PR TITLE
Mpatel31415/fix for missing tokens per sec

### PR DESCRIPTION
**Pull request by mpatel31415**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/approved via a Github issue? 
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

Fixes https://github.com/Lightning-AI/lightning-thunder/issues/1267

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Yes 🙃

### CODE REVIEW

**Review by t-vi — APPROVED** [review]
Thank you @mpatel31415

File: thunder/benchmarks/benchmark_litgpt.py [hunk 1]
```diff
@@ -819,8 +819,9 @@ def benchmark_main(return_metrics_as_json=False, json_path="", **kwargs) -> None
 
         print(f"Average iter time: {benchmark.perf_metrics['average_iter_time']:.2f} ms")
         print(f"Memory used: {benchmark.perf_metrics['memory_used_GB']:.02f} GB")
-        print(f"Tokens/s: {benchmark.perf_metrics['tokens_per_sec']:.02f}")
-        print(f"Tokens/s/GPU: {(benchmark.perf_metrics['tokens_per_sec']/world_size):.02f}")
+        if "tokens_per_sec" in benchmark.perf_metrics:
+            print(f"Tokens/s: {benchmark.perf_metrics.get['tokens_per_sec']:.02f}")
```
Comment (IvanYashchuk) [inline thunder/benchmarks/benchmark_litgpt.py:823]: `.get` is subscriptable. Maybe a better version would be to condition on tokens_per_sec not to be None `if benchmark.perf_metrics['tokens_per_sec'] is not None:`?

**Review by IvanYashchuk — COMMENTED** [review]
@mpatel31415, could you please take a look? I don't think the merged code is working.

File: thunder/benchmarks/benchmark_litgpt.py (hunk 1 above)
Comment (mpatel31415) [reply thunder/benchmarks/benchmark_litgpt.py:823]: Actually benchmark.perf_metrics.get['tokens_per_sec'] returns None if "tokens_per_sec" is not in benchmark.perf_metrics and then there is problem with formatting: `:.02f`.

I can change it to (to be more readable and cover case with tokens_per_sec present but being None)
```
if tokens_per_sec := benchmark.perf_metrics.get('tokens_per_sec') is not None:
            print(f"Tokens/s: {tokens_per_sec:.02f}")
            print(f"Tokens/s/GPU: {(tokens_per_sec/ world_size):.02f}")
```
WDYT?

**Pull request merged**